infoschema/tables.go

```diff
@@ -1368,6 +1371,14 @@ var tableDeadlocksCols = []columnInfo{
 	{name: "TRX_HOLDING_LOCK", tp: mysql.TypeLonglong, size: 21, flag: mysql.NotNullFlag, comment: "The transaction ID (start ts) of the transaction that's currently holding the lock"},
 }
 
+var tableDataLockWaitsCols = []columnInfo{
```

Comment:
Is it possible to know how long does the transaction wait for a lock?

Reply:
This infomation is supposed to be provided on `TIDB_TRX`.
eg.
```sql
select NOW() - START_TIME from INFORMATION_SCHEMA.TIDB_TRX where XXX;
```